### PR TITLE
new "security" section in the docs : Changing/encrypting the DNS

### PR DESCRIPTION
**Pull request by evenbecker**
I have written a tutorial to change the DNS server and encrypt the DNS.

### CODE REVIEW

File: _docs/hw_accel.md [hunk 1]
```diff
@@ -47,4 +47,4 @@ $ LIBGL_DEBUG=verbose glxinfo | grep libgl
 ```
 
 More info see:
-[https://en.wikipedia.org/wiki/Graphics_processing_unit#GPU_accelerated_video_decoding_and_encoding]
+[https://en.wikipedia.org/wiki/Graphics_processing_unit#GPU_accelerated_video_decoding_and_encoding)
```
Comment (demmm) [inline _docs/hw_accel.md]: In Markdown links are done with `[title](url)`
The above will not display a link

File: _docs/changing_dns.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+date: '2024-05-01 10:06:56 +0200'
+date_gmt: '2024-05-01 10:06:56 +0200'
+categories: []
+tags: []
+comments: []
+---
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+**Changing/encrypting the DNS**
```
Comment (demmm) [inline _docs/changing_dns.md]: Docs layout automatically adds the title:
https://github.com/KaOSx/kaosx.github.io/blob/master/_layouts/docs.html#L22

Don't repeat that here

File: _docs/changing_dns.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+{: .offset}
+
+DNS over TLS (DoT) is a system-wide DNS encryption technique. It is an efficient protection against network attackers.
+Government surveillance also involves collecting and analyzing DNS queries. A VPN subscription is probably the best protection but it is not free. In this case, DoT can be a good and free alternative. The DNS provided by your ISP may not support DoT, so Quad9 DNS is a good choice.
+
+To use DoT, add this line in /etc/systemd/resolved.conf :
+```
+DNSOverTLS=yes
+```
+
```
Comment (demmm) [inline _docs/changing_dns.md:74]: If you plan to do more website work, it is best to set up a local jekyll instance, so you can see what you are working on.
To have a new doc show on the website, it needs to be added to the proper section in https://github.com/KaOSx/kaosx.github.io/blob/master/_data/docs.yml

For reference, see the correction done for you:
https://github.com/KaOSx/kaosx.github.io/commit/fd3af45bc570897095abcaa4570ec9238c780f5a

Comment (evenbecker) [reply _docs/changing_dns.md:74]: Ok. I don't plan to do more website work.
 If I understand correctly, I have to add a line 'changing_dns' in docs.yml, right?

Comment (demmm) [reply _docs/changing_dns.md:74]: If not planning on more, then I will do the additions, just focus on getting the .md files corrected.

Comment (evenbecker) [reply _docs/changing_dns.md:74]: I think I am done with the corrections. I have a last question : Do I have to sync my fork everytime something has changed in KaOSx:master?

File: _docs/hw_accel.md [hunk 4]
```diff
@@ -47,4 +47,4 @@ $ LIBGL_DEBUG=verbose glxinfo | grep libgl
 ```
 
 More info see:
-[https://en.wikipedia.org/wiki/Graphics_processing_unit#GPU_accelerated_video_decoding_and_encoding)
+[https://en.wikipedia.org/wiki/Graphics_processing_unit#GPU_accelerated_video_decoding_and_encoding](https://en.wikipedia.org/wiki/Graphics_processing_unit#GPU_accelerated_video_decoding_and_encoding)
```
Comment (demmm) [inline _docs/hw_accel.md]: That still won't show a link, again, you need to use `[title](url)`

Comment (evenbecker) [reply _docs/hw_accel.md]: ok, I have corrected.

**Pull request merged**